task2-bash/copy_prj_script.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
+Sourse="../task1-simple-program"
```

Comment:
A common practice for shell scripts is to create environment variables in upper case.
It will simplify life for people who will review or enhance your code.